<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>PTZcameraForm</class>
<widget class="QWidget" name="PTZcameraForm">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>731</width>
   <height>772</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>PTZcameraForm</string>
 </property>
 <widget class="QWidget" name="centralwidget" native="true">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>761</height>
   </rect>
  </property>
  <widget class="QLabel" name="testPicture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="filterPicture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>491</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Up">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вверх</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Left">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Влево</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Right">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вправо</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Down">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вниз</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Disconnect">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отключиться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectAutomatic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вкл. Автом. Подстройку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisconnectAutomatic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Откл. Автом. Подстройку</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>20</width>
     <height>761</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>366</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="TetraAddressThree">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TetraAddressTwo">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TetraAddressFour">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести ip-адрес камеры</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TetraAddressOne">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести порты камеры:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Login">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>520</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести логин камеры:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести пароль камеры:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Сервер:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortRTSP">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rtsp:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="invertBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Инвертировать оси камеры</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>640</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tilt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>640</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Pan">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>660</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RightUp">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вправо/Вверх</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftUp">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Влево/Вверх</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftDown">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Влево/Вниз</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightDown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вправо/Вниз</string>
   </property>
  </widget>
 </widget>
 <widget class="QMenuBar" name="menubar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>21</height>
   </rect>
  </property>
 </widget>
 <widget class="QStatusBar" name="statusbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3</width>
    <height>18</height>
   </rect>
  </property>
 </widget>
</widget>
<resources/>
<connections/>
</ui>
